src/metakb/normalizers.py

```diff
@@ -222,3 +232,86 @@ def get_regulatory_approval_extension(
                 )
 
         return regulatory_approval_extension
+
+
+class NormalizerName(str, Enum):
+    """Constrain normalizer CLI options."""
+
+    GENE = "gene"
+    DISEASE = "disease"
+    THERAPY = "therapy"
+
+    def __repr__(self) -> str:
+        """Print as simple string rather than enum wrapper, e.g. 'gene' instead of
+        <NormalizerName.GENE: 'gene'>.
+
+        Makes Click error messages prettier.
+
+        :return: formatted enum value
+        """
+        return f"'{self.value}'"
+
+
+def check_normalizers(
+    db_url: str | None, normalizers: Iterable[NormalizerName] | None = None
+) -> bool:
+    """Perform basic health checks on the gene, disease, and therapy normalizers.
+
+    Uses the internal health check methods provided by each.
+
+    :param db_url: optional designation of DB URL to use for each. Currently only
+        works for DynamoDB. If not given, normalizers will fall back on their own
+        env var/default configurations.
+    :param normalizers: names of specific normalizers to check (check all if empty/None)
+    :return: True if all normalizers pass all checks, False if any failures are
+        encountered.
+    """
+    success = True
+    normalizer_map = {
+        NormalizerName.DISEASE: create_disease_db,
+        NormalizerName.THERAPY: create_therapy_db,
+        NormalizerName.GENE: create_gene_db,
+    }
+    if normalizers:
+        normalizer_map = {k: v for k, v in normalizer_map.items() if k in normalizers}
+    for name, create_db in normalizer_map.items():
+        db = create_db(db_url)
+        try:
+            schema_initialized = db.check_schema_initialized()
+            if not schema_initialized:
+                _logger.warning(
+                    "Schema for %s normalizer appears incomplete or nonexistent.",
+                    name.value,
+                )
+                success = False
+                continue
+            tables_populated = db.check_tables_populated()
+            if not tables_populated:
+                _logger.warning(
+                    "Tables for %s normalizer appear to be unpopulated.", name.value
+                )
+                success = False
+        except Exception as e:
+            _logger.error(
+                "Encountered exception while checking %s normalizer: %s", name.value, e
+            )
+            success = False
+    return success
+
+
+def update_normalizer(normalizer: NormalizerName, db_url: str | None) -> None:
```

Comment:
Would we want to move the AWS checks here? I don't think MetaKB should be used to update any of the AWS normalizer instances